exporter/awskinesisexporter/internal/compress/noop_compression.go

```diff
@@ -26,3 +26,5 @@ func (n noop) Write(p []byte) (int, error) {
 func (n noop) Flush() error {
 	return nil
 }
+
+func (n *noop) Close() error { return n.Close() }
```

Comment:
Aa, this should return nil here I believe.

Reply:
great call, thanks! 
fixed it and pushed